tests/pipedrive/test_pipedrive_source.py

```diff
@@ -323,10 +323,12 @@ def test_resource_settings() -> None:
 
     assert source.resources["custom_fields_mapping"].write_disposition == "replace"
 
+    s = source.discover_schema()
+
     for rs_name in resource_names - {"custom_fields_mapping"}:
         rs = source.resources[rs_name]
         assert rs.write_disposition == "merge"
-        assert rs.table_schema()["columns"]["id"]["primary_key"] is True
```

Comment:
It doesn't look like this method even exists

Reply:
huh! how it was working before. anyway: thanks for fixing!